### DISCUSSION TITLE
Explanation of POSTaggerME tags

### DISCUSSION THREAD

**Discussion by hamiltonjose** [sergey-tihon/OpenNLP.NET#22] [General]
Hello guys,
I'm using the POSTaggerME to retrieve tags from the token array which is working just fine (thank you). 
However, I cannot understand the meaning of some tags. 
A couple examples of tags that I am getting from the mentioned class:
PROP
ADV
PUNC

The tags above are not part of the "official" list of POS: https://www.ling.upenn.edu/courses/Fall_2003/ling001/penn_treebank_pos.html

Is there an alternative mapping between the two? Would you be able to clarify?

Thanks.

**Comment by sergey-tihon** [comment]
TBH, I do not know )

The trained model comes from the Apache/Java project. Official documentation https://opennlp.apache.org/docs/1.8.0/manual/opennlp.html#tools.postagger has a line with broken link

> The tag set used by the english pos model is the [Penn Treebank tag set](http://www.cis.upenn.edu/~treebank/).


I may only guess that they may come from universal POS tags - https://universaldependencies.org/u/pos/index.html
